### DISCUSSION TITLE
How to commit PR, on `GitHubs Action  Documents` what i been translate?

### DISCUSSION THREAD

**Discussion by Yjiwan** [community/community#41698] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I wonder who translate contents. 

Because, translated only Title which you can see in below Screenshot. 

So i want to add Korean contents for not only who could read korea documents but who knew Korean language.

How can i commit PR or forked that repo?

![Screen Shot 2022-12-14 at 4 03 33 PM](https://user-images.githubusercontent.com/85088046/207528276-b51fcccf-1498-4345-aeaf-0b36181183ce.png)

Here is link:

[Understanding GitHub Actions](https://docs.github.com/ko/actions/learn-github-actions/understanding-github-actions)

**Comment by ghostinhershell** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @Yjiwan! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

With that being said, thanks so much for showing interest in contributing to GitHub's Docs! You can learn more about the Docs contribution workflow: from opening an issue, creating a PR, reviewing, and merging the PR, [here](https://github.com/github/docs/blob/main/CONTRIBUTING.md).

I've gone ahead and moved this to the correct category for you. Please let us know if you have any additional questions about contributing to our Docs. Cheers!
